assets/dev/scss/admin/_promotion.scss

```diff
@@ -0,0 +1,108 @@
+@mixin margin( $top: inherit, $right: inherit, $bottom: inherit, $left: inherit ) {
```

Comment:
I think the video screenshots shouldn't be included in this PR.
Flaky test fixes should be done in separate PRs.